src/backend/src/controllers/projects.controllers.ts

```diff
@@ -26,12 +26,53 @@ export default class ProjectsController {
     }
   }
 
+  // static async createProject(req: Request, res: Response, next: NextFunction) {
+  //   try {
+  //     const user: User = await getCurrentUser(res);
+  //     const { crId, carNumber, name, summary, teamId } = req.body;
+
+  //     const createdWbsNumber: WbsNumber = await ProjectsService.createProject(user, crId, carNumber, name, summary, teamId);
+
+  //     return res.status(200).json(wbsPipe(createdWbsNumber));
+  //   } catch (error: unknown) {
+  //     next(error);
+  //   }
+  // }
   static async createProject(req: Request, res: Response, next: NextFunction) {
     try {
       const user: User = await getCurrentUser(res);
-      const { crId, carNumber, name, summary, teamId } = req.body;
+      const {
+        name,
+        crId,
+        carNumber,
+        teamIds,
+        budget,
+        summary,
+        projectLeadId,
+        projectManagerId,
+        links,
+        goals,
+        features,
+        otherConstraints,
+        rules
+      } = req.body;
 
-      const createdWbsNumber: WbsNumber = await ProjectsService.createProject(user, crId, carNumber, name, summary, teamId);
+      const createdWbsNumber: WbsNumber = await ProjectsService.createProject(
+        user,
+        crId,
+        carNumber,
+        name,
+        summary,
+        teamIds,
+        budget,
```

Comment:
budget through projectManagerId should be optional, so do || null like we do for edit project

Reply:
you'll need to update the route and service method as well

Reply:
So in the prisma create call for budget, for example, would it be called like

```ts
const createdWbsElement = await prisma.wBS_Element.create({
      data: {
        ...
        project: {
          create: {
            // like this
            budget: budget ?? undefined,
    });
```

Reply:
if that works yeah, I don't know that we've done a create with optional fields like this before so there's not really a precedent